src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockCollection/blockCollection.scss

```diff
@@ -42,7 +42,7 @@
 
 .selectMultipleButtonContainer {
     display: flex;
-    justify-content: end;
```

Comment:
was this change intentional?

Reply:
yes autoprefixer did complain that `flex-end` is supported better then `end` and has same effect and did output a warning message about this line.